assets/javascripts/modules/checkout/regex.js

```diff
@@ -1,7 +1,9 @@
 define([], function () {
+    'use strict';
+
     return {
         isNumber: function isNumber(s) {
```

Comment:
For these kinds of tests, it might be worth loading `https://github.com/lodash/lodash-amd` (which is very useful to have around anyway) as it includes https://lodash.com/docs#isNumber . The AMD build allows you to include single functions so you only get the minimum required in your build. cc/ @tudorraul